### ISSUE TITLE
notice

### ISSUE THREAD

**Issue by Sonic1paRadise** [rafaelvcaetano/melonDS-android#855]
## Read before you proceed (and delete this section before posting)
Only post the issue if you believe there is a problem with the emulator. If you have a question, feature request or other generic topic, you are encouraged to open a [Discussion](https://github.com/rafaelvcaetano/melonDS-android/discussions) instead. If you are not sure whether you are dealing with an issue or not, don't worry and post it wherever you feel is the most appropriate place.

Issues asking about online Wi-Fi will most likely be ignored and deleted since that feature is not yet implemented. You can, however, open a discussion if you believe you have valuable information to add regarding this topic.
**********************

* **Device model:** Redmi 6A
* **Android version:** Android 9 Pie

**Problem:**  
Write a small description of the problem.

**Steps to reproduce (if applicable):**
1. Write a list of steps
2. Required for the problem
3. To appear

Add screenshots if the issue can be visualized.
why you cannot research more in nintendo ds emulation and optimize the emulation without jit like drastic

**Comment by ASGAMING009** [comment]
Redmi 6A is a a low end device ( i have that device) soo if you are saying That its laging. It will be cause Its a 32bit and have a low end processor and only has 2 gb ram.

**Comment by Sonic1paRadise** [comment]
my having 4 gb ram 16gb rom no lag on drastic lag on melon ds
why you cannot research what kind of optimization drastic uses

чт, 10 бер. 2022 р. о 06:18 ASGAMING009 ***@***.***> пише:

**Comment by maxdlpee** [comment]
Emulators like Drastic are focused on performance, while melonDS is focused on accuracy, so it is only natural that the latter has worse performance on your device than the former (specially considering the fact that it is a low-end device)

**Comment by Sonic1paRadise** [comment]
Yes you partially right. But if you just research the optimisation and fix
the bugs for accuracy, this would be smooth and accurate. Thanks

чт, 10 мар. 2022 г., 22:36 maxdlpee ***@***.***>:

**Comment by Sonic1paRadise** [comment]
P.S my phone has 2ghz cpu. He cannot lag. My pc haves the same too and was
running melonds perfectly.

чт, 10 мар. 2022 г., 6:18 ASGAMING009 ***@***.***>:

**Comment by rafaelvcaetano** [comment]
Sorry for not replying in this topic. I'll try to clarify some misconceptions.

A PC processor and a mobile processor are very different. They might have similar clock speeds and core count, but a PC CPU might use around 100W while a high-end mobile CPU doesn't even reach 10W. Since there are no miracles, you could never hope to achieve the same performance in both.

Regarding the comparisons to DraStic, it is as maxdlpee said. melonDS is more focused in accuracy instead of performance. Increasing performance is not a question of choosing how much we want but rather a bi-product of how the development of the whole project is approached. The emulator is not slow because we chose to. And finding out how to improve its performance is not something we can simply search on Google. These are niche products for which there are not a lot of people that know how to do it and there's no official documentation to guide us. If something doesn't work properly, you have to dedicate hours to find out what's wrong and sometimes guess how the original hardware might have worked. Even I don't have such abilities. This project is only possible thanks to the devs of the PC version. I just make their work run on the Android platform.

Don't take this the wrong way. I'm not bashing you. Just showing you that things are not so simple and that we are all here working to make the best emulator we can.
